.travis.yml

```diff
@@ -16,22 +16,21 @@ install:
   - docker exec bblfshd bblfshctl driver install bblfsh/python-driver:$BBLFSH_PYTHON_VERSION
   - wget https://github.com/bblfsh/client-python/releases/download/v2.2.1/protobuf-python_3.4.1-1_amd64.deb
   - sudo dpkg -i protobuf-python_3.4.1-1_amd64.deb
+  - if [[ ! -z "$TRAVIS_TAG" ]]; then sed -i -e "s/^VERSION\s*=\s*\"[^\"]\+\"/VERSION = \"${TRAVIS_TAG#v}\"/g" setup.py; fi
   - pip3 install --upgrade pip
   - pip3 install -r requirements.txt
   - python3 setup.py --getdeps --log
   - pip3 install . --upgrade
-  - cd bblfsh && python3 -m unittest discover
-  - if [[ -z "$TRAVIS_TAG" ]]; then exit 0; fi
-  - if [[ $TRAVIS_PYTHON_VERSION != '3.6' ]]; then exit 0; fi # disable double uploads to pypi
-  - echo "[distutils]" > .pypirc
-  - echo "index-servers = " >> .pypirc
-  - echo "  pypi" >> .pypirc
-  - echo "[pypi]" >> .pypirc
-  - echo "username=$PYPI_USER" >> .pypirc
-  - echo "password=$PYPI_PASS" >> .pypirc
-  - cd ..
-  - HOME=. python setup.py sdist upload
+  - cd bblfsh && python3 -m unittest discover && cd ..
 script:
   - python3 setup.py build_ext -i
-notifications:
-  email: false
+after_success:
+  - if [[ -z "$TRAVIS_TAG" ]]; then exit 0; fi
+  - if [[ $TRAVIS_PYTHON_VERSION != '3.6' ]]; then exit 0; fi # disable double uploads to pypi
+  - echo "[distutils]" > $HOME/.pypirc
+  - echo "index-servers = " >> $HOME/.pypirc
+  - echo "  pypi" >> $HOME/.pypirc
+  - echo "[pypi]" >> $HOME/.pypirc
+  - echo "username=$PYPI_USER" >> $HOME/.pypirc
+  - echo "password=$PYPI_PASS" >> $HOME/.pypirc
+  - python setup.py sdist upload
```

Comment:
```suggestion
  - python setup.py sdist upload
```

Reply:
Sorry, I don't see any change on this line.

Reply:
Assuming it's okay. Let me know if you want me to change something else.

Reply:
It was just adding the missing LF at EOF. Cosmetic, not crucial.